clr-installer/main.go

```diff
@@ -176,6 +189,13 @@ func main() {
 		defer func() { _ = os.Remove(cf) }()
 	}
 
+	if filepath.Ext(cf) == ".json" {
+		cf, err = model.JSONtoYAMLConfig(cf)
+		if err != nil {
+			fatal(err)
+		}
+	}
```

Comment:
We should add log.Info() message here that the JSON was automatically converted to YAML.

Reply:
Since the JSONtoYAMLConfig() is called twice in the code, added the log.Info() message inside this function along with the corresponding fmt.Println message.